### ISSUE TITLE
error when add component to cms page

### ISSUE THREAD

**Issue by vincent2090311** [magesuite/magesuite#23]
![Screen Shot 2019-10-09 at 11 34 20](https://user-images.githubusercontent.com/8340835/66451968-ecc07b00-ea88-11e9-8f1e-95cb6d0e9520.png)
When I tried to select a component, I got an error like the screenshot

**Comment by krzksz** [comment]
Hello @vincent2090311, please make sure you have theme-creativeshop or any of its children set as a theme for global context in your shop.
![obraz](https://user-images.githubusercontent.com/829189/66458698-af9dbe00-ea73-11e9-95aa-30467d374f52.png)

**Comment by vincent2090311** [comment]
@krzksz 
is the frontend theme take effect to admin UI?

**Comment by krzksz** [comment]
Yes, frontend theme can influence some of the components you are able to configure within content constructor. You may find the current options inside `theme-creativeshop/src/etc/view.xml` under `MageSuite_ContentConstructor` var.

**Comment by krisdante** [comment]
@krzksz this needs to be documented somewhere. Not that obvious thing when somebody would like to implement own FE

**Comment by michalzimniak** [comment]
I dont have Global without other names entry on this list, but the same error if want to add component to cms page. How to add it?

![image](https://user-images.githubusercontent.com/7513244/81782873-c10fe280-94fa-11ea-9c75-99449c4c9c7d.png)

**Comment by koria** [comment]
> I dont have Global without other names entry on this list, but the same error if want to add component to cms page. How to add it?
> 
> ![image](https://user-images.githubusercontent.com/7513244/81782873-c10fe280-94fa-11ea-9c75-99449c4c9c7d.png)


I have the same issue.
Have you solved this problem?

**Comment by koria** [comment]
It's work when i add a new store view

**Comment by p-makowski** [comment]
This problem still exists in "creativestyle/magesuite":"6.x-dev"

![image](https://user-images.githubusercontent.com/6819904/100025292-c672b180-2de8-11eb-9ba7-f238c5980527.png)

**Comment by thomas7467** [comment]
Hi,
Just for information, I had the same issue after using data migration tool and importing global, Main Website and 3 Store view with "creativestyle/magesuite": "^7.0.0".
New M2 install and magesuite tests with content editor worked fine. Issues appeared for me after 1.9 migration.  
I can confirm thet the issue occurs with Global theme not configured with Theme Creativshop, that I had leaved "blank" firstly.
After configuring Theme Creativeshop for Global and Child Theme for Website and Store View, adding components in Component Editor works again.
My current configuration ; M 2.4.1, PHP 7.4.3.
Best regards
Thomas
